test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Kindly consider adding this file the gitignore as suggested by the project requirements. This is to ensure that unwanted files are not committed to the github repository.